src/status_im/utils/ethereum/core.cljs

```diff
@@ -82,4 +83,10 @@
   (.sendTransaction (.-eth web3) (clj->js params) cb))
 
 (def default-transaction-gas (money/bignumber 21000))
-(def default-gas-price (money/->wei :gwei 21))
\ No newline at end of file
+(def default-gas-price (money/->wei :gwei 21))
+
+(defn- estimated-gas [symbol]
```

Comment:
Maybe `estimate-gas` and make public? Seems useful

Reply:
:+1: Should be public actually it's used elsewhere.